pkgs/ifupdown-ng.nix

```diff
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  libbsd,
+  iproute2,
+  ...
+}:
+stdenv.mkDerivation {
+  pname = "ifupdown-ng";
+  version = "0.12.1";
+
+  src = fetchFromGitHub {
+    owner = "ifupdown-ng";
+    repo = "ifupdown-ng";
+    rev = "e305296b3d23e56bba92c504e030d8e4b91db403";
+    sha256 = "1zf04waaklqvq4ywxyjswwpxpmq7jrbjkw3yd8aapxfj20gkph56";
+  };
+  buildInputs = [libbsd iproute2];
+  patches = [
+    ./ifupdown-fix-path.patch
+  ];
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp ifupdown $out/bin
+    cp ifup $out/bin
+    cp ifdown $out/bin
+    cp ifquery $out/bin
+    cp ifparse $out/bin
+    cp ifctrstat $out/bin
```

Comment:
Binaries should prefer `install` over `cp`, and a `for` loop can save a few lines here.